### DISCUSSION TITLE
Someone pls help meee :(

### DISCUSSION THREAD

**Discussion by Arthura0121** [3kh0/ext-remover#316] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
So, my Chromebook updated by itself and I can't rollback on crosh because this is a school computer and I hate GoGuardian so can someone please help me? I know, I know, that this exploit only works for chrome versions 105 and lower but I'm on 107 and I really want to turn these extensions off. Thank you, have a nice day!! :)) @3kh0 @qDread-dev @Synaptic149 @yeeteeyt @CountBleck

**Comment by CountBleck** [comment, marked as answer]
The only thing you can do is use a proxy or downgrade. crosh has nothing to do with downgrading by the way. There aren't many options left.

-------- Original Message --------
On Dec 12, 2022, 17:53, Arthura0121 < ***@***.***> wrote:
So, my Chromebook updated by itself and I can't rollback on crosh because this is a school computer and I hate GoGuardian so can someone please help me? I know, I know, that this exploit only works for chrome versions 105 and lower but I'm on 107 and I really want to turn these extensions off. Thank you, have a nice day!! :)) @3kh0 @qDread-dev @Synaptic149 @yeeteeyt @CountBleck
—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you were mentioned.￼Message ID: ***@***.***>
[ { ***@***.***": "http://schema.org", ***@***.***": "EmailMessage", "potentialAction": { ***@***.***": "ViewAction", "target": "[https://github.com](<a href=)/3kh0/ext-remover/discussions/316">https://github.com/3kh0/ext-remover/discussions/316", "url": "[https://github.com](<a href=)/3kh0/ext-remover/discussions/316">https://github.com/3kh0/ext-remover/discussions/316", "name": "View Discussion" }, "description": "View this Discussion on GitHub", "publisher": { ***@***.***": "Organization", "name": "GitHub", "url": "https://github.com" } } ]

**Reply by Aka-but-nice** [reply]
Yes, it does, you can use the "rollback" command to revert to old versions of chrome!

**Reply by Arthura0121** [reply]
the thing is the school blocked the rollback command on the school computers

**Comment by qDread-dev** [comment]
Please don't ping me for no reason :)

**Reply by 3kh0** [reply]
https://github.com/3kh0/ext-remover#for-v106-and-up it is here!

**Reply by Arthura0121** [reply]
My school disabled inspect, is there another way?

**Reply by 3kh0** [reply]
@Arthura0121 I am afraid not

**Reply by Arthura0121** [reply]
Ok... but the thing is that the Chromebooks lets us update it like every month. Does the exploit work on version 108?

**Comment by Aka-but-nice** [comment]
Yo, tell me what you did on crosh

**Comment by CountBleck** [comment]
Don't ping me for no reason either

**Comment by CountBleck** [comment]
`rollback` sounds like a Developer Mode thing, so I doubt anyone would be able to use it.
-------- Original Message --------

**Comment by Aka-but-nice** [comment]
No, it ain't but schools can prevent you from using it. That's why I said try it because maybe your school is more retarded than mines.
